<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>121</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>552</width>
    <height>121</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>552</width>
    <height>121</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWebView" name="webNews">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="url">
    <url>
     <string>http://dotacash.com/api/news.php</string>
    </url>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>9</y>
     <width>247</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Forum Account Name:</string>
      </property>
      <property name="buddy">
       <cstring>txtUsername</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="txtUsername">
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Forum Account Pass: </string>
      </property>
      <property name="buddy">
       <cstring>txtPassword</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="txtPassword">
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnLogin">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Don't have an account?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRegister">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.dotacash.com/forums/register&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;&amp;lt;Click to Register&amp;gt;&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>webNews</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtUsername</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtPassword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
